### ISSUE TITLE
AU M6 CS not being triggered

### ISSUE THREAD

**Issue by EpicTaru** [LegionDark/Issues#1393]
**Date & Time**: 
03/06/2017 @ 1400 EST

**FFXI Client Version (use `/ver`)**: 
30170204_1

**Server's Expected Client Version matches, yes/no? (use `$ecv`)**: 
Yes

**Character Name**: 
EpicTaru

**Nation**: 
San d'Oria

**Job(level)/Sub Job(level)**: 
99THF/49DNC

**NPC or Monster or item Name**: 
N/A

**Zone name**: 
Ru'Lude Gardens

**Coordinates (use `$where`)**: 
X:0 Y:-1 Z:53 (141)

**ffxiah.com link (for items issues only)**: 
N/A

**Multi-boxing? (multiple clients on same connection)**: 
No

**Steps To Reproduce / any other info**: 
Zone into Ru'Lude Gardens (either from HP crystals or from Upper Jeuno), walk into palace, CS should trigger upon entrance into palace but the CS is never triggered.

**Comment by Heepster03** [comment]
@TeoTwawki , I believe this might have you written all over it.

I checked the mission they were on and then I performed the checkvar for AhtUrganStatus and it was at 2. Not sure why this was as it should've been 0 and then turned to a 1 after the quest started. So i reset it to 0 and had them talk to Halver again, it set itself to 1 which is what is needed for the CS in Rulude. However, they are not getting said CS even though they are on the correct mission and the AhtUrganStatus is at 1 (which is what the mission is calling for.)
